docs/api/classes/_lit_element_.cssresult.html

```diff
@@ -153,7 +153,7 @@ <h3>constructor</h3>
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/Polymer/lit-element/blob/557de8c/src/lib/css-tag.ts#L25">src/lib/css-tag.ts:25</a></li>
+									<li>Defined in <a href="https://github.com/PolymerLabs/polymer-lit/blob/3e18aa4/src/lib/css-tag.ts#L25">src/lib/css-tag.ts:25</a></li>
```

Comment:
Weird that these changed to an old URL...

Reply:
Oh, weird. I noticed the SHA change but not the repo change. @sorvell are you working from an old git client with an old origin?

Reply:
Ah, the docs generator uses the git origin to make the links? Makes sense.

@sorvell You can do `git remote -v` to check.

Probably just need to do this and re-generate:
```
git remote set-url origin https://github.com/Polymer/lit-element.git
```

Reply:
Good catch. Fixed.